### DISCUSSION TITLE
Feature request: Skyblock Active Minion's on Islands

### DISCUSSION THREAD

**Discussion by jqwhal** [HypixelDev/PublicAPI#552] [Ideas]
Hello, I would love to have some way of finding out active island minions through the API, afaik this is stored in island offline data, but there must be some not too complex way of grabbing this data. e.g. each time a player changes what item is in the minion/picks up/places down a minion. Please let me know if/why I'm wrong about this.

**Comment by MattTheCuber** [comment]
This would be very useful!
